tests/test_Restart.cpp

```diff
@@ -221,9 +221,24 @@ data::Solution mkSolution( int numCells ) {
     return sol;
 }
 
-Opm::SummaryState sim_state()
+Opm::SummaryState sim_state(const Opm::Schedule& sched)
 {
     auto state = Opm::SummaryState{TimeService::now()};
+    for (const auto& well : sched.getWellsatEnd()) {
+        for (const auto& connection : well.getConnections()) {
+            state.update_conn_var(well.name(), "CPR", connection.global_index() + 1, 100);
+            if (well.isInjector()) {
+                state.update_conn_var(well.name(), "COIR", connection.global_index() + 1, 100);
+                state.update_conn_var(well.name(), "CGIR", connection.global_index() + 1, 100);
+                state.update_conn_var(well.name(), "CWIR", connection.global_index() + 1, 100);
+            } else {
+                state.update_conn_var(well.name(), "COPR", connection.global_index() + 1, 100);
+                state.update_conn_var(well.name(), "CGPR", connection.global_index() + 1, 100);
+                state.update_conn_var(well.name(), "CWPR", connection.global_index() + 1, 100);
+            }
```

Comment:
If you're going to add new values anyway, may I ask that you make the values distinct too?  That way it'll be easier to track down the source if a test fails.